Marlin/src/HAL/HAL_STM32/timers.h

```diff
@@ -71,8 +71,35 @@
     #define TEMP_TIMER 10
   #endif
 
+#elif defined(STM32F4xx)
+
+  #define HAL_TIMER_RATE (F_CPU/2) // frequency of timer peripherals
+
+
+  #ifndef STEP_TIMER
+    #define STEP_TIMER 6
+  #endif
+
+  #ifndef TEMP_TIMER
+    #define TEMP_TIMER 14
+  #endif
+
+#elif defined(STM32F7xx
+
+  #define HAL_TIMER_RATE (F_CPU/2) // frequency of timer peripherals
+
+  #ifndef STEP_TIMER
+    #define STEP_TIMER 6
+  #endif
+
+  #ifndef TEMP_TIMER
+    #define TEMP_TIMER 14
+  #endif
+
 #endif
```

Comment:
I think those two #elif statements can be merged into one by adjusting the first check

Reply:
I'd rather leave them separate.  It could make things easier in the future if we run into F407 vs. F7xx issues.

On the other hand I still find the STM32 world confusing.  I'll be happy to follow the advice of the experts.

Reply:
I don't think there are any F7 devices actually working from here right now, so I'd say it is only coincidence that they are the same right now.